demo/api/src/db/fixtures/fixtures.console.ts

```diff
@@ -291,8 +289,6 @@ export class FixturesConsole {
 
         await this.dependenciesService.createViews();
 
-        await this.productsFixtureService.generate();
```

Comment:
Please don't delete the products fixtures.

Reply:
omg, how has this happend 🙈